user.go

```diff
@@ -36,3 +36,65 @@ func (s *System) GetUser() ([]types.User, error) {
 
 	return user, nil
 }
+
+// GetUserByIDName returns a specific user based on it's user id
+func (s *System) GetUserByIDName(userID string, username string) (*types.User, error) {
+	if userID == "" && username == "" {
+
+		return nil, errors.New("user name or ID is mandatory, please enter a valid value")
+
+	} else if userID != "" {
+		path := fmt.Sprintf("/api/instances/User::%v",
+			userID)
+		user := &types.User{}
+		err := s.client.getJSONWithRetry(
+			http.MethodGet, path, nil, &user)
+		if err != nil {
+			return nil, err
+		}
+
+		return user, nil
+
+	} else {
+		allUsers, err := s.GetUser()
+		if err != nil {
+			return nil, err
+		}
+
+		for _, u := range allUsers {
```

Comment:
i think instead of u if it is user it will be nice

Reply:
Done